packages/pug-code-gen/index.js

```diff
@@ -311,10 +316,17 @@ Compiler.prototype = {
     }
 
     if (debug && node.debug !== false && node.type !== 'Block') {
+      if (typeof node.line !== 'number') {
+        throw new Error(
+          'node.line is not a valid number.'
+        );
+      }
```

Comment:
```suggestion
    if (debug && node.debug !== false && node.type !== 'Block' && typeof node.line === 'number') {
```
isnt should better?

Reply:
It would, but that meant that it’s checking for all three instead of checking for the required two and failing silently. Throwing an error is better as it can alert the user that there may be a pollution.

Reply:
Hmmm i understand